avatarly.gemspec

```diff
@@ -3,7 +3,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency('rfc822')
 
   s.name        = "avatarly"
-  s.version     = "1.1.0"
+  s.version     = "1.1.1"
```

Comment:
I guess you should bump up minor version instead of patch since this pull request adds new feature.
http://semver.org/

It should be 
```ruby
 s.version     = "1.2.0"
```